### PR TITLE
Inga dev

### PR DESCRIPTION
**Pull request by ingarun**
Switched out align_si.sh for align_fmllr.sh before sat training and updated a string in data prep file according to the name change in the train field in the metadata (training -> train)

### CODE REVIEW

File: s5_base/run.sh [hunk 1]
```diff
@@ -98,21 +103,31 @@ if [ $stage -le 4 ]; then
...
 fi
 
 if [ $stage -le 5 ]; then
     echo "Preparing a pruned trigram language model"
     mkdir -p data/log
-    $train_cmd --mem 24G data/log/make_LM_3g.log local/make_LM.sh --order 3 --carpa false \
-      --min1cnt 20 --min2cnt 10 --min3cnt 2 $lm_train data/lang $localdict/lexicon.txt data \
-      || error 1 "Failed creating a pruned trigram language model";
-
+    $train_cmd --mem 24G data/log/make_LM_3g.log \
```
Comment (Daviderikmollberg) [inline s5_base/run.sh:115]: Shouldn't the mem parameter be located in cmd.sh?

File: s5_base/run.sh [hunk 2]
```diff
@@ -98,21 +103,31 @@ if [ $stage -le 4 ]; then
...
+    local/make_LM.sh \
+    --order 3 --carpa false \
+    --min1cnt 20 --min2cnt 10 --min3cnt 2 \
+    $lm_train data/lang \
+    $localdict/lexicon.txt data \
+    || error 1 "Failed creating a pruned trigram language model";
+    
     echo "Preparing an unpruned 4g LM"
-    $train_cmd --mem 32G data/log/make_LM_4g.log local/make_LM.sh --order 4 --carpa true \
```
Comment (Daviderikmollberg) [inline s5_base/run.sh:113]: Same here,

File: s5_base/run.sh [hunk 3]
```diff
@@ -125,62 +140,107 @@ fi
...
     echo "Train LDA + MLLT + SAT"
     steps/train_sat.sh --cmd "$train_cmd --mem 4G"  4000 40000 data/train \
-      data/lang exp/tri2_ali exp/tri3
+    data/lang exp/tri2_ali exp/tri3
 fi
 
 # Tri3 decoding
 if [ $stage -le 10 ]; then
     echo "Triphone tri3 decoding"
-    $train_cmd --mem 4G exp/tri3/log/mkgraph.log utils/mkgraph.sh data/lang_3g exp/tri3 exp/tri3/graph
```
Comment (Daviderikmollberg) [inline s5_base/run.sh:167]: I see it's different for almost each call. 

In my subword script, I have different 

cmd calls for different actions but I guess that is just preferance

**Review by Daviderikmollberg — COMMENTED** [review]
Looks good, almost only formatting changes

File: s5_base/run.sh (hunk 1 above)
Comment (ingarun) [reply s5_base/run.sh:115]: The problem with these is that the memory varies with how big the input is so I would then have to create two new commands in the cmd.sh and I wasn't sure whether I should do that

**Pull request merged**